packages/firestore/exp/test/deps/dependencies.json

```diff
@@ -0,0 +1,15 @@
+{
+    "foo": {
+        "dependencies": [
+            "bar",
+            "foo"
+        ],
+        "sizeInBytes": 67
+    },
+    "bar": {
+        "dependencies": [
+            "bar"
+        ],
+        "sizeInBytes": 39
```

Comment:
this is per function? seems like a pain to always update

Reply:
It is auto-updated via the script, but the script is not run automatically. I'll send this to @Feiyang1 and we can ponder how we can automate this step.